docs/build/developer-reference/l1-l2-interop.md

```diff
@@ -25,6 +25,16 @@ L1 to L2 communication is governed by the [`IZkSync.sol`](https://github.com/mat
 - If you prefer to learn-by-doing, the [cross chain governance tutorial](../tutorials/smart-contract-development/cross-chain-tutorial.md) is a practical example of layer interoperability.
   :::
 
+#### L1 to L2 transaction filtering
+
+A hyperchain operator can filter L1→L2 transactions coming through the Diamond proxy. It is achieved by having an
+address of the special contract, responsible for transaction filtering, in the state transition storage. If the filterer
+exists, it is being called in the [Mailbox facet](https://github.com/matter-labs/era-contracts/blob/dev/l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol#L243)
```

Comment:
These links are for the `dev` branch. As these are not live, do we want to hold off on merging these changes until `1.5.0` is live?

Reply:
Yeah, it makes sense to wait for `1.5.0` before merging. I'll keep track of it and ping you when it's ready to be merged